keyboards/eek/info.json

```diff
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "eek!",
+    "url": "http://www.keyboard-layout-editor.com/#/gists/3e4d0cd1534dd4e4b9f25cf2b36d0d63",
+    "maintainer": "klackygears",
+    "width": 13,
+    "height": 6.25,
+    "layouts": {
+        "LAYOUT": {
+            "layout": [{"label":"E", "x":2, "y":0}, {"label":"I", "x":8, "y":0}, {"label":"W", "x":1, "y":0.38}, {"label":"R", "x":3, "y":0.38}, {"label":"U", "x":7, "y":0.38}, {"label":"O", "x":9, "y":0.38}, {"label":"T", "x":4, "y":0.5}, {"label":"Y", "x":6, "y":0.5}, {"label":"Q", "x":0, "y":0.88}, {"label":"P", "x":10, "y":0.88}, {"label":"D", "x":2, "y":1.0}, {"label":"K", "x":8, "y":1.0}, {"label":"S", "x":1, "y":1.3800000000000001}, {"label":"F", "x":3, "y":1.3800000000000001}, {"label":"J", "x":7, "y":1.3800000000000001}, {"label":"L", "x":9, "y":1.3800000000000001}, {"label":"G", "x":4, "y":1.5}, {"label":"H", "x":6, "y":1.5}, {"label":"A", "x":0, "y":1.8800000000000001}, {"label":";", "x":10, "y":1.8800000000000001}, {"label":"C", "x":2, "y":2}, {"label":",", "x":8, "y":2}, {"label":"X", "x":1, "y":2.38}, {"label":"V", "x":3, "y":2.38}, {"label":"M", "x":7, "y":2.38}, {"label":".", "x":9, "y":2.38}, {"label":"B", "x":4, "y":2.5}, {"label":"N", "x":6, "y":2.5}, {"label":"Z", "x":0, "y":2.88}, {"label":"/", "x":10, "y":2.88}, {"label":"Ctrl", "x":2, "y":3.25}, {"label":"Shift", "x":8, "y":3.25}, {"label":"Lower", "x":3, "y":3.38}, {"label":"Raise", "x":7, "y":3.38}, {"label":"Space", "x":4, "y":3.5}, {"label":"BSpc", "x":6, "y":3.5}]
+        }
+    }
+}
```

Comment:
```suggestion
{
    "keyboard_name": "eek!",
    "url": "http://www.keyboard-layout-editor.com/#/gists/3e4d0cd1534dd4e4b9f25cf2b36d0d63",
    "maintainer": "klackygears",
    "width": 13,
    "height": 6.25,
    "layouts": {
        "LAYOUT": {
            "layout": [
                {"x": 0, "y": 0.88},
                {"x": 1, "y": 0.38},
                {"x": 2, "y": 0},
                {"x": 3, "y": 0.38},
                {"x": 4, "y": 0.5},

                {"x": 6, "y": 0.5},
                {"x": 7, "y": 0.38},
                {"x": 8, "y": 0},
                {"x": 9, "y": 0.38},
                {"x": 10, "y": 0.88},

                {"x": 0, "y": 1.88},
                {"x": 1, "y": 1.38},
                {"x": 2, "y": 1},
                {"x": 3, "y": 1.38},
                {"x": 4, "y": 1.5},

                {"x": 6, "y": 1.5},
                {"x": 7, "y": 1.38},
                {"x": 8, "y": 1.0},
                {"x": 9, "y": 1.38},
                {"x": 10, "y": 1.88},

                {"x": 0, "y": 2.88},
                {"x": 1, "y": 2.38},
                {"x": 2, "y": 2},
                {"x": 3, "y": 2.38},
                {"x": 4, "y": 2.5},

                {"x": 6, "y": 2.5},
                {"x": 7, "y": 2.38},
                {"x": 8, "y": 2},
                {"x": 9, "y": 2.38},
                {"x": 10, "y": 2.88},

                {"x": 2, "y": 3.25},
                {"x": 3, "y": 3.38},
                {"x": 4, "y": 3.5},

                {"x": 6, "y": 3.5},
                {"x": 7, "y": 3.38},
                {"x": 8, "y": 3.25}
            ]
        }
    }
}
```
Not quite, there were still some rounding errors, and the keys were out of order (you can also check this in Configurator by turning on Fast Input and assigning keycodes). I also cleaned it up a little :)

Reply:
Oh! Thanks!